### PR TITLE
Py313rc3

### PR DESCRIPTION
**Pull request by cbouss**
python 3.13rc3

**Destination channel:** ad-testing/label/py313

### Links

- https://anaconda.atlassian.net/browse/PKG-5732
- https://github.com/python/cpython/tree/v3.13.0rc3

### CODE REVIEW

File: recipe/build_base.sh [hunk 1]
```diff
@@ -247,6 +254,12 @@ _common_configure_args+=(--enable-loadable-sqlite-extensions)
 _common_configure_args+=(--with-tcltk-includes="-I${PREFIX}/include")
 _common_configure_args+=("--with-tcltk-libs=-L${PREFIX}/lib -ltcl8.6 -ltk8.6")
 _common_configure_args+=(--with-platlibdir=lib)
+# TODO build libmpdec as a conda package, https://www.bytereef.org/mpdecimal/
+_common_configure_args+=(--with-system-libmpdec=no)
```
Comment (cbouss) [inline recipe/build_base.sh:258]:
```suggestion
_common_configure_args+=(--with-system-libmpdec=yes)
```

**Review by ryanskeith — COMMENTED** [review]
LGTM. Check note about lib2to3 reference.

**Review by danpetry — APPROVED** [review]
We did a live review of this and I'm happy with the changes. Mostly they've been validated via conda-forge and tested by the HPP team. An exception is the free-threading variant on windows, but the test suite has been run. The changes also mostly affect the free-threading variant which is presented as experimental anyway. I initially brought up the idea of further testing but for the reasons above I'm good with what's been done.

**Review by ryanskeith — APPROVED** [review]
Approved after small cleanup.

**Comment by cbouss** [comment]
> We did a live review of this and I'm happy with the changes. Mostly they've been validated via conda-forge and tested by the HPP team. An exception is the free-threading variant on windows, but the test suite has been run. The changes also mostly affect the free-threading variant which is presented as experimental anyway. I initially brought up the idea of further testing but for the reasons above I'm good with what's been done.

I'll merge this PR as is. The result goes to ad-testing/label/py313. sitecustomize.py may not be the right solution for users having a system-wied sitecustomize.py. This means the freethreading variant will need more investigation and will not be enabled on the official release PR. Work will be tracked here: https://anaconda.atlassian.net/browse/PKG-5855

**Comment by danpetry** [comment]
ok. Thanks

**Pull request merged**